src/dns_private.tf

```diff
@@ -102,3 +102,21 @@ resource "azurerm_private_dns_zone" "privatelink_azurecr_pagopa" {
 
   tags = var.tags
 }
+
+# MongoDB
+resource "azurerm_private_dns_zone" "privatelink_mongo_cosmos_azure_com" {
```

Comment:
private link for all envs? also for DEV?

Reply:
disable for DEV